src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/OptionsTests.cs

```diff
@@ -734,5 +735,26 @@ public static void ConverterRead_VerifyInvalidTypeToConvertFails()
                 }
             }
         }
+
+        [Fact]
+        public static void GetConverterRootsBuiltInConverters()
```

Comment:
Should these go in the "CustomConverterTests" namespace instead of the tests for `JsonSerializerOptions`?